brewtils/plugin.py

```diff
@@ -388,19 +401,27 @@ def _shutdown(self):
         self.logger.debug("Successfully shutdown plugin {0}".format(self.unique_name))
 
     def _create_standard_consumer(self):
-        return RequestConsumer(amqp_url=self.instance.queue_info['url'],
-                               queue_name=self.instance.queue_info['request']['name'],
-                               on_message_callback=self.process_request_message,
-                               panic_event=self.shutdown_event, thread_name='Request Consumer',
-                               max_concurrent=self.max_concurrent)
+        return RequestConsumer(
+            thread_name='Request Consumer',
+            connection_info=self.amq_params,
+            amqp_url=self.instance.queue_info.get('url', None),
```

Comment:
Out of curiosity, what happens if both the `url` and `connection` are provided by the server? Might be worth a comment.

Reply:
`connection` takes priority since it's the only one that allows SSL.